src/main/java/core/basesyntax/service/impl/CsvReportWriterService.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportWriterService;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class CsvReportWriterService implements ReportWriterService {
+    private static final String REPORT_PATH = "./src/main/resources/report.csv";
```

Comment:
pass path to file as a parameter of method below to make this service more flexible